packages/blocks/github-pr-overview/src/entity-aggregations.ts

```diff
@@ -1,261 +1,297 @@
-import {
-  BlockProtocolAggregateEntitiesFunction,
-  BlockProtocolAggregateEntitiesResult,
-} from "blockprotocol";
+import { GraphBlockHandler } from "@blockprotocol/graph";
 import { uniqBy } from "lodash";
-import { BlockState } from "./app";
 
 import {
-  GithubIssueEvent,
-  GithubPullRequest,
+  GithubIssueEventEntityType,
+  GithubPullRequestEntityType,
   PullRequestIdentifier,
-  GithubReview,
+  GithubReviewEntityType,
   isDefined,
+  GITHUB_ENTITY_TYPES,
 } from "./types";
 
 const ITEMS_PER_PAGE = 100;
 
-export const getPrs = (
-  githubPullRequestTypeId: string,
-  aggregateEntities?: BlockProtocolAggregateEntitiesFunction,
-  accountId?: string | null,
-  pageNumber?: number,
-  selectedPullRequest?: PullRequestIdentifier,
-): Promise<BlockProtocolAggregateEntitiesResult<GithubPullRequest> | void> => {
-  if (!aggregateEntities) {
-    return new Promise<void>(() => {});
-  }
-
-  const res = aggregateEntities({
-    accountId,
-    operation: {
-      entityTypeId: githubPullRequestTypeId,
-      pageNumber,
-      itemsPerPage: ITEMS_PER_PAGE,
-      multiFilter: {
-        operator: "AND",
-        filters:
-          selectedPullRequest !== undefined
-            ? [
-                {
-                  field: "html_url",
-                  operator: "CONTAINS",
-                  value: `${selectedPullRequest.repository}/pull/${selectedPullRequest.number}`,
-                },
-              ]
-            : [],
+export const getGithubEntityTypes = async (
+  aggregateEntityTypes: GraphBlockHandler["aggregateEntityTypes"],
+  numPages: number = 5,
```

Comment:
It seems weird to me to keep the `numPages` argument if we're just using it as a multiplier for some const value. I think we should either swap to having a `num_items` parameter, or to making `itemsPerPage` a top-level const on its own.

Reply:
I agree. Will update